boards/arm/nucleo_f411re/nucleo_f411re_defconfig

```diff
@@ -12,6 +12,10 @@ CONFIG_STM32_ARM_MPU_ENABLE=y
 # enable USART2 - passthrough to STLINK v2 connector
 CONFIG_UART_STM32_PORT_2=y
 
+# enable I2C1
+CONFIG_I2C=y
+CONFIG_I2C_STM32_V1=y
+
```

Comment:
Please also enable I2C2 as documented in peripheral mapping

Reply:
See comment above.

Reply:
I was referring the i2c you specify in the board documentation, so you should activate it in the defconfig accordingly

Reply:
I'm happy to change the board config to not select I2C_1 but to add CONFIG_I2C_1=y and CONFIG_I2C_2=y to the defconfig would be different to every other board. I'm happy to do whatever you want but it seems odd.

Reply:
The idea was to match to rst doc in the defconfig, but not enforce it in the board config